<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApTrendSelect</class>
 <widget class="QWidget" name="ApTrendSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="bnAp1">
     <property name="minimumSize">
      <size>
       <width>672</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вакуум-апарат №1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="bnAp2">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>708</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вакуум-апарат №2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="bnAp3">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>708</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вакуум-апарат №3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="bnAp4">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>708</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Загальне обладнання 1-го продукту</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>132</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>132</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>613</width>
       <height>107</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>613</width>
       <height>107</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
